openvpn/config.go

```diff
@@ -39,6 +40,11 @@ func (c *Config) SetManagementSocket(socketAddress string) {
 	c.setFlag("management-client")
 }
 
+func (c *Config) SetScripts(runtimeDirectory string) {
+	c.setParam("up", filepath.Join(runtimeDirectory, "update-resolv-conf"))
```

Comment:
Call this script relativelly as `update-resolv-conf`:
- Will be searched in neighbourhood
- Will be searched in PATH